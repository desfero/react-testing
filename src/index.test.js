import React from 'react';

import './index';
import {render} from 'react-dom';
jest.mock('react-dom');

describe('index', () => {
    it('should render component', async () => {
        expect(render.mock.calls[0][0]).toMatchSnapshot();
    });
});