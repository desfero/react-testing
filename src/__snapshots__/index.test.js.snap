// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`index should render component 1`] = `
<Provider
  store={
    Object {
      "dispatch": [Function],
      "getState": [Function],
      "replaceReducer": [Function],
      "subscribe": [Function],
      Symbol(observable): [Function],
    }
  }
>
  <BrowserRouter>
    <div>
      <Unknown />
    </div>
  </BrowserRouter>
</Provider>
`;
